todo_app/tests/test_view_model.py

```diff
@@ -0,0 +1,58 @@
+from operator import length_hint
+from todo_app.view_model import ViewModel
+import pytest
+
+@pytest.fixture
+def test_cards():
+
+    cards = [{'id': '1', 'status': 'todo', 'name': 'testcard1'},
+             {'id': '2', 'status': 'todo', 'name': 'testcard2'},
+             {'id': '3', 'status': 'doing', 'name': 'testcard3'},
+             {'id': '4', 'status': 'doing', 'name': 'testcard4'},
+             {'id': '5', 'status': 'doing', 'name': 'testcard5'},
+             {'id': '6', 'status': 'done', 'name': 'testcard6'}]
```

Comment:
This passes the tests, but your actual code assumes that the items passed to the ViewModel are of your `Item` class type, so better to use those here:
```suggestion
    cards = [
            Item('1', 'todo', 'testcard1'),
            Item('2', 'todo', 'testcard2'),
            Item('3', 'doing', 'testcard3'),
            Item('4', 'doing', 'testcard4'),
            Item('5', 'doing', 'testcard5'),
            Item('6', 'done', 'testcard6'),
    ]
```

Note this does mean your ViewModel & tests will need to access statuses like `card.status` rather than `card['status']`